lib/src/parse/stylesheet.dart

```diff
@@ -1443,8 +1458,9 @@ abstract class StylesheetParser extends Parser {
   /// [start] should point before the `@`.
   @protected
   SupportsRule supportsRule(LineScannerState start) {
+    whitespace(allowNewlines: true);
```

Comment:
Because this is a plain-CSS rule that may eventually have no special parsing, we shouldn't allow newlines here.